os/arch/arm/src/armv8-m/mpu.h

```diff
@@ -160,6 +160,11 @@
 #define MPU_RASR_AP_RORO      (6 << MPU_RASR_AP_SHIFT)	/* P:RO   U:RO   */
 #define MPU_RASR_XN             (1 << 28)	/* Bit 28: Instruction access disable */
 
+#ifdef CONFIG_MPU_STACK_OVERFLOW_PROTECTION
+#define MPU_REGION_ADDRESS_ALIGNMENT 32	/* MPU regions must be 32 byte address aligned */
```

Comment:
1. Comment can be modified for better clarity . MPU Regions must be size aligned though minimum size possible is 32 bytes.. 
2. MPU_REGION_ADDRESS_ALIGNMENT is quiet generic. IMO, a specific one like MPU_REGION_STACKGAURD_ADDRESS_ALIGNMENT  would be good

Reply:
1. It is 32 byte address aligned along with 32 byte size aligned for MPU.
2. It is generic macro for MPU, so removed out of stack overflow define